### PR TITLE
issue-22835-feedback-on-new-findings-on-dropdowns-keyboard-navigation

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
### Proposed Changes
- https://github.com/dotcms/core/issues/22835#issuecomment-1260078706

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/view/components/dot-dialog/dot-dialog.component.ts [hunk 1]
```diff
@@ -172,17 +172,10 @@ export class DotDialogComponent implements OnChanges {
...
-
-            case 'Enter':
-                this.acceptAction();
-                break;
-
-            default:
-                break;
+        if (event.code === 'Escape') {
+            this.cancelAction();
+        } else if (event.code === 'Enter' && (event.metaKey || event.altKey)) {
```
Comment (rjvelazco) [inline core-web/apps/dotcms-ui/src/app/view/components/dot-dialog/dot-dialog.component.ts:177]: Why are the `$event.metaKey` and `$event.altKey` needed?

**Review by rjvelazco — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/view/components/dot-dialog/dot-dialog.component.ts (hunk 1 above)
Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/view/components/dot-dialog/dot-dialog.component.ts:177]: metaKey is for Mac OS 

altKey is for Windows/Linux

File: core-web/apps/dotcms-ui/src/app/view/components/dot-form-dialog/dot-form-dialog.component.spec.ts [hunk 2]
```diff
@@ -93,6 +105,14 @@ describe('DotFormDialogComponent', () => {
             expect(component.save.emit).toHaveBeenCalledWith(event);
         });
 
+        // TODO: find a way to dispatch the event and that it gets caught by the listener
+        xit('should have emit save event on CMD + ENTER keys', () => {
+            const saveButton = de.query(By.css('#testInput'));
+            dispatchKeydownEvent(saveButton.nativeElement, 'enter', true);
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/view/components/dot-form-dialog/dot-form-dialog.component.spec.ts:111]: I think the problem might be related with that you are sending the event from a button but the listener in the component is tied to the document.

Also can you please check those listener? I think the subscribres are not being unsubscribe.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/view/components/dot-dialog/dot-dialog.component.ts (hunk 1 above)
Comment (rjvelazco) [reply core-web/apps/dotcms-ui/src/app/view/components/dot-dialog/dot-dialog.component.ts:177]: Cool!

**Review by hmoreras — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**